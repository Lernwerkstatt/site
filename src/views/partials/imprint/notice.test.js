const notice = require("./notice.hbs");

describe("notice.hbs", () => {
  it("should match the snapshot", () => {
    expect(
      notice({
        imprint_text: "Lorem ipsum"
      })
    ).toMatchSnapshot();
  });
});
