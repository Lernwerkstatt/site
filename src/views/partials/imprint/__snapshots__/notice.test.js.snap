// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`notice.hbs should match the snapshot 1`] = `
"<div class="container">
  <div class="col-sm-8 ml-n3 mb-5">
    i18n
  </div>
</div>
"
`;
